homeassistant/components/shelly/binary_sensor.py

```diff
@@ -48,13 +53,30 @@
     ),
 }
 
+REST_SENSORS = {
+    ("cloud"): RestAttributeDescription(
+        name="Cloud",
+        device_class=DEVICE_CLASS_CONNECTIVITY,
+        path="cloud/connected",
+    ),
+    ("fwupdate"): RestAttributeDescription(
+        name="Firmware update (release)",
+        device_class=DEVICE_CLASS_SAFETY,
```

Comment:
I don't think 'safety' fits here. According to the documentation "safety: on means unsafe, off means safe".

Reply:
IMHO, old firmware may be unsafe

Reply:
May be but doesn't have to.

Reply:
So what class do you suggest ? I still think that safety is the best choice

Reply:
No device class.

Reply:
Ok, then 

https://materialdesignicons.com/icon/chip
or
https://materialdesignicons.com/icon/update

as icon ?

Reply:
We can use `update` icon or no icon at all. Home Assistant will then use the default icon.

Reply:
I applied the suggestion, even if we do't have an aggreement if we leave or not the sensor.